pkg/virt-operator/resource/generate/rbac/cluster.go

```diff
@@ -274,6 +275,7 @@ func newEditClusterRole() *rbacv1.ClusterRole {
 					"virtualmachineinstances/console",
 					"virtualmachineinstances/vnc",
 					"virtualmachineinstances/vnc/screenshot",
+					"virtualmachineinstances/vsock",
```

Comment:
Same here. Should probably be removed too.

Reply:
Done